calibre fails to build (QT incompatibility?)

I think this happened in the last couple of days.  It's not clear to me how to address it, though.

> ####### Building headless QPA plugin #######
> Project MESSAGE: This project is using private headers and will therefore be tied to this specific Qt module build version.
> Project MESSAGE: Running this project against other versions of the Qt modules may crash at any arbitrary point.
> Project MESSAGE: This is not a bug, but a result of using Qt internals. You have been warned!
> make: *** No rule to make target '/tmp/nix-build-calibre-2.28.0.drv-0/qmake-calibre-2.28.0/lib/libQt5PlatformSupport.a', needed by '/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/src/calibre/plugins/libheadless.so'.  Stop.
> Error while executing: make -j4
> 
> Traceback (most recent call last):
>   File "setup.py", line 99, in <module>
>     sys.exit(main())
>   File "setup.py", line 85, in main
>     command.run_all(opts)
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/__init__.py", line 181, in run_all
>     self.run_cmd(self, opts)
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/__init__.py", line 175, in run_cmd
>     self.run_cmd(scmd, opts)
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/__init__.py", line 178, in run_cmd
>     cmd.run(opts)
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/extensions.py", line 414, in run
>     self.build_headless()
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/extensions.py", line 550, in build_headless
>     self.check_call([make] + ['-j%d'%(cpu_count or 1)])
>   File "/tmp/nix-build-calibre-2.28.0.drv-0/calibre-2.28.0/setup/extensions.py", line 486, in check_call
>     subprocess.check_call(*args, **kwargs)
>   File "/nix/store/fzsbv209kna7bz6c89g9h1b2q3hmkaaz-python-2.7.9/lib/python2.7/subprocess.py", line 540, in check_call
>     raise CalledProcessError(retcode, cmd)
> subprocess.CalledProcessError: Command '['make', '-j4']' returned non-zero exit status 2
> builder for ‘/nix/store/2rfhnx9d63d313j5hi2cxz25z5c1ri9a-calibre-2.28.0.drv’ failed with exit code 1
> error: build of ‘/nix/store/2rfhnx9d63d313j5hi2cxz25z5c1ri9a-calibre-2.28.0.drv’ failed

--- a/pkgs/development/libraries/qt-5/5.4/setup-hook.sh
+++ b/pkgs/development/libraries/qt-5/5.4/setup-hook.sh
@@ -20,9 +20,11 @@ addQtModule() {
             fi
         fi
 
-        if [[ -n $qtSubmodule ]] && [[ -d "$1/lib" ]]; then
+        if [[ -d "$1/lib" ]]; then
             @lndir@/bin/lndir -silent "$1/lib" "$qtOut/lib"
-            find "$1/lib" -printf 'lib/%P\n' >> "$qtOut/nix-support/qt-inputs"
+            if [[ -n $qtSubmodule ]]; then
+                find "$1/lib" -printf 'lib/%P\n' >> "$qtOut/nix-support/qt-inputs"
+            fi
         fi
     fi
 }

